### PR TITLE
Update dashboard alex

### LINKED ISSUE #271: Format astronomy page of dashboard
As an amateur astronomer I want easy access to astronomy data in a clean readable format so that I can stargaze effectively

- Format dashboard page for astronomy data

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #271

I have Incorporated the skyplots into the dashboard

### CODE REVIEW

File: dashboard/aurora_map.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def create_aurora_map() -> plt.figure:
     return fig
 
 
-def create_visibilty_map() -> plt.figure:
+def create_visibility_map() -> plt.figure:
```
Comment (CalebMathers) [inline dashboard/aurora_map.py:217]: thanks

File: dashboard/azimuth_plot.py [hunk 2]
```diff
@@ -13,6 +15,24 @@
...
+
+def get_bodies() -> list:
+    '''Returns a list of all bodies'''
+    query = '''SELECT body_name FROM body'''
+    with get_connection() as conn:
+        with conn.cursor() as cur:
+            cur.execute(query)
+            data = list(cur.fetchall())
+    return data
+
```
Comment (olliejth) [inline dashboard/azimuth_plot.py:35]: Could these be done with the same connection?

File: dashboard/azimuth_plot.py [hunk 3]
```diff
@@ -135,15 +155,35 @@ def animate_skyplot(celestial_df: pd.DataFrame, region: str):
...
+    with get_connection() as conn:
+        with conn.cursor() as cur:
+            cur.execute(query,(body,))
+            data = cur.fetchone()
+    return data
 
 if __name__ == '__main__':
     load_dotenv()
-    make_sky_plot('Scotland', dt(2024, 10, 25, 0, 0, 0))
+    make_sky_plot('Scotland', dt(2024, 10, 17, 0, 0, 0))
```
Comment (CalebMathers) [inline dashboard/azimuth_plot.py:188]: FREEEEEDOMMMM

File: dashboard/dashboard.py [hunk 4]
```diff
@@ -439,9 +462,39 @@ def get_current_datetime() -> datetime:
                st.error(f'Error adding subscriber: {e}')
 
 
-elif page == 'Test':
-   conn = connect_to_db()
-
-   bodies_df = load_celestial_body_information(conn)
-
-   print(bodies_df.head(10))
+elif page == 'Astronomy':
```
Comment (CalebMathers) [inline dashboard/dashboard.py:465]: Good way to split up the pages

File: dashboard/nasa_pipeline.py [hunk 5]
```diff
@@ -121,4 +131,4 @@ def get_sunset_times(region: str) -> tuple:
     logger = logging.getLogger(__name__)
     logging.basicConfig(level=logging.INFO)
     load_dotenv()
-    print(get_sunset_times('Staffordshire'))
+    print(get_moon_phase())
```
Comment (olliejth) [inline dashboard/nasa_pipeline.py:134]: I see that this print statement is necessary for the pipeline.

File: dashboard/nasa_pipeline.py [hunk 6]
```diff
@@ -23,9 +23,19 @@ def get_image_of_the_day() -> str:
...
             cur.execute(query)
             data = cur.fetchone()
-            print(data)
     return data
 
+def get_moon_phase() -> str:
+    '''Returns todays moon phase'''
+    query = """SELECT image_url FROM image
+    WHERE image_name = 'moon_phase' AND
+    image_date = CURRENT_DATE"""
```
Comment (CalebMathers) [inline dashboard/nasa_pipeline.py:32]: Nice simple query, no need to overcomplicate it, thanks to our streamlined database no doubt

**Review by CalebMathers — APPROVED** [review]

**Review by olliejth — APPROVED** [review]
Good changes.

**Pull request merged**